src/main/java/com/ericsson/gerrit/plugins/eiffel/GerritModule.java

```diff
@@ -47,6 +48,10 @@ protected void configure() {
         bind(LifecycleListener.class).annotatedWith(UniqueAnnotations.create())
                                      .to(MessageQueueHandler.class);
 
+        // Register change listener that will send messages
+        DynamicSet.bind(binder(), EventListener.class).to(ChangeMergedEventListener.class);
+        DynamicSet.bind(binder(), EventListener.class).to(PatchsetCreatedEventListener.class);
```

Comment:
Cant you add this to a method with a describing methodname instead of adding a comment?

Reply:
Donel

Reply:
The comment itself was never removed, please remove them as well :) Otherwise it looked great!

Reply:
Done.